e2e/screenshot.ts

```diff
@@ -3,11 +3,14 @@ import * as path from 'path';
 import {browser} from 'protractor';
 
 const OUTPUT_DIR = './screenshots/';
+const HEIGHT = 788;
```

Comment:
is there some significance to these numbers? isn't the standard screen size usually 1024x768